src/cvode/cvode_bbdpre.c

```diff
@@ -237,27 +239,33 @@ int CVBBDPrecInit(void *cvode_mem, sunindextype Nlocal,
   pdata->rpwsize = 0;
   pdata->ipwsize = 0;
   if (cv_mem->cv_tempv->ops->nvspace) {
-    N_VSpace(cv_mem->cv_tempv, &lrw1, &liw1);
+    SUNCheckCallLastErrNoRet(N_VSpace(cv_mem->cv_tempv, &lrw1, &liw1));
     pdata->rpwsize += 3*lrw1;
     pdata->ipwsize += 3*liw1;
   }
   if (pdata->rlocal->ops->nvspace) {
-    N_VSpace(pdata->rlocal, &lrw1, &liw1);
+    SUNCheckCallLastErrNoRet(N_VSpace(pdata->rlocal, &lrw1, &liw1));
     pdata->rpwsize += 2*lrw1;
     pdata->ipwsize += 2*liw1;
   }
   if (pdata->savedJ->ops->space) {
     flag = SUNMatSpace(pdata->savedJ, &lrw, &liw);
+    SUNCheckCallNoRet(flag);
+    SUNCheck(flag == SUN_SUCCESS, flag);
```

Comment:
This is the first file where I've seen subsequent calls to both `SUNCheckCallNoRet` and `SUNCheck` following a single call.  Should one of these be removed?  I'll note that this same pattern occurs two more times below.